lib/cProfile.ml

```diff
@@ -8,6 +8,8 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
+let enable_profile = false
```

Comment:
This flag seems useless, since prof_table is extended without looking at it. IMO is the better to just remove it.

Reply:
Sounds good.

Reply:
Actually that would break plugin cprofile code , as explained in the comment.

Reply:
I think that you can run init_profile always, and if someone happens to register something in the table one gets the final print.

If you don't purge the flag, then in order to profile a function I need to add the wrapper and set the flag. Also, it lets one commit wrapper code as long as he does not put true here, and nobody will notice.

IMO the API should be:
- you register a wrapper, you get the prints
- you don't, you don't get the prints

The boolean is just a nanosecond optimization we can remove.

Reply:
So, what's the conclusion?

Reply:
Ping. This is really a detail IMO.

Reply:
if you think it is an irrelevant detail, then go on and merge